app/getUsers.tsx

```diff
@@ -0,0 +1,14 @@
+async function getData() {
+  try {
+    const response = await fetch('https://jsonplaceholder.typicode.com/users');
+    const data = response.json();
```

Comment:
@ninman27 We can return the response.json directly instead of creating the data variable that is only used once.